yt/yt/server/query_tracker/proxy_service.cpp

```diff
@@ -0,0 +1,268 @@
+#include "private.h"
```

Comment:
This logic is completely moved from client_queries_impl.cpp with small structural changes